SampleApplications/SDK/Opc.Ua.Client/Session.cs

```diff
@@ -2198,7 +2198,15 @@ public ReferenceDescriptionCollection FetchReferences(NodeId nodeId)
                 catch (Exception ex)
                 {
                     Utils.Trace("Create session failed with client certificate NULL. " + ex.Message);
-                    successCreateSession = false;
+                    if (ex.Message == "BadTooManySessions")
```

Comment:
This should be something like this:

                    ServiceResultException srEx = ex as ServiceResultException;

                    if (srEx != null && srEx.StatusCode == StatusCodes.BadTooManySessions)
                    {
                        throw ex;
                    }
                    else
                    {
                        Utils.Trace("Create session failed with client certificate NULL. " + ex.Message);
                        successCreateSession = false;
                    }